src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -452,10 +459,39 @@ class ClusterTaskManager : public ClusterTaskManagerInterface {
 
   const int64_t sched_cls_cap_max_ms_;
 
-  /// Metrics collected since the last report.
-  uint64_t metric_tasks_queued_;
-  uint64_t metric_tasks_dispatched_;
-  uint64_t metric_tasks_spilled_;
+  struct InternalStats {
+    /// Number of tasks that are spilled to other
+    /// nodes because it cannot be scheduled locally.
+    size_t metric_tasks_spilled = 0;
```

Comment:
Prefer using int64_t instead of size_t. size_t should only be used for size in bytes or offsets: https://www.embedded.com/why-size_t-matters/

Reply:
Good to know. It is fixed!